cmd/zpool/zpool_main.c

```diff
@@ -314,10 +316,10 @@ get_usage(zpool_help_t idx)
 		    "[-R root] [-F [-n]]\n"
 		    "\t    <pool | id> [newpool]\n"));
 	case HELP_IOSTAT:
-		return (gettext("\tiostat [-c CMD] [-T d | u] [-ghHLpPvy] "
-		    "[[-lq]|[-r|-w]]\n"
-		    "\t    [[pool ...]|[pool vdev ...]|[vdev ...]] "
-		    "[interval [count]]\n"));
+		return (gettext("\tiostat [[[-c [script1,script2,...]"
```

Comment:
You can run with just -c to get the list of the scripts:
```
$ zpool status -c
Current scripts in /usr/local/etc/zfs/zpool.d:

  enc             Show disk enclosure w:x:y:z value.
  encdev          Show the /dev/sg* device for the enclosure associated with the disk slot.
  fault_led       Show the value of the disk enclosure slot fault LED.
  iostat-1s       Do a single 1-second iostat sample and show values.
  iostat-10s      Do a single 10-second iostat sample and show values.
  label           Show filesystem label.
  locate_led      Show the value of the disk enclosure slot locate LED.
  lsblk           Show the disk size, vendor, model number, and serial number.
  model           Show disk model number.
  serial          Show disk serial number.
  ses             Show disk's enclosure, enclosure dev, slot number, and fault/locate LED values.
  size            Show the disk capacity.
  slaves          Show device mapper slave devices.
  slot            Show disk slot number as reported by the enclosure.
  upath           Show the underlying path for a device.
  vendor          Show the disk vendor.
  iostat          Show iostat values since boot (summary page).

```